RELEASE/scripts/sl_ascend.ash

```diff
@@ -843,13 +843,18 @@ item[monster] catBurglarHeistDesires()
 	if(needFood) neededTrimmers++;
 	if(needJar) neededTrimmers++;
 	if(needInit) neededTrimmers++;
-	if ((catBurglarHeistsLeft() >= 2) && (neededTrimmers > 0))
+	if ((my_level() >= 8) && (catBurglarHeistsLeft() >= 2) && (neededTrimmers > 0))
 	{
 		wannaHeists[$monster[bearpig topiary animal]] = $item[rusty hedge trimmers];
 		wannaHeists[$monster[elephant (meatcar?) topiary animal]] = $item[rusty hedge trimmers];
 		wannaHeists[$monster[spider (duck?) topiary animal]] = $item[rusty hedge trimmers];
 	}
 
+	if(get_property("questL11Shen") == "finished" && internalQuestStatus("questL11Ron") == 1 && catBurglarHeistsLeft() >= 2)
```

Comment:
Why bother checking if `questL11Shen` is finished?

Reply:
same reason, log spamming with `sl_debug=True`. Happy to revert, I was planning on removing that debug logging in a few revisions anyway.

Reply:
Nah, all good.